README.md

```diff
@@ -75,6 +75,47 @@ but not limited to:
 * Sigfox
 * LoRaWAN
 
+## GETTING RIOT
```

Comment:
Wouldn't "Getting RIOT" a part of how to Get Started?

Reply:
"Getting started" in this file is basically just a collection of links to the real documentation, so this did not seem fitting to me. Unless you are asking to add this to the main page of the documentation.

Reply:
I mean the "Getting Started" section. Seems a bit short, and getting the code is part of getting started to me

Reply:
It links to more detailed guides. I have doubts, the links get drowned in the information provided here if it is not its own section.

Reply:
Ok